tokio/CHANGELOG.md

```diff
@@ -1,3 +1,93 @@
+# 1.26.0 (March 1st, 2023)
+
+### Fixed
+
+- sync: don't leak tracing spans in mutex guards ([#5469])
+- sync: drop wakers after unlocking the mutex in Notify ([#5471])
+- sync: drop wakers outside lock in semaphore ([#5475])
+- macros: fix empty `join!` and `try_join!` ([#5504])
+
+### Added
+
+- fs: add `fs::try_exists` ([#4299])
+- net: add types for named unix pipes ([#5351])
+- sync: add `MappedOwnedMutexGuard` ([#5474])
+
+### Documented
+
+- task: clarify what happens to spawned work during runtime shutdown ([#5394])
+- task: clarify `process::Command` docs (#5406) ([#5413])
```

Comment:
Just the PR here.

Reply:
Oh, missed this, the commit had that in the header line!

Reply:
Fixed